test/standard/1.8/parameter.schema.json

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
don't commit it here

Reply:
Do I let the configuration to this file in the configs/test.exs?